lib/active_model/serializer/logging.rb

```diff
@@ -0,0 +1,32 @@
+module ActiveModel
+  class Serializer
+    module Logging
+      extend ActiveSupport::Concern
+
+      included do
+        cattr_accessor(:logger) do
+          ActiveSupport::TaggedLogging.new(ActiveSupport::Logger.new(STDOUT))
+        end
+      end
+
+      class LogSubscriber < ActiveSupport::LogSubscriber
+        def render(event)
+          serializer = event.payload[:serializer]
+          adapter = event.payload[:adapter]
+          duration = event.duration.round(2)
+          logger.tagged('AMS') do
+            info do
+              "Rendered #{serializer.name} with #{adapter.class} (#{duration}ms)"
```

Comment:
I do not understand this line:

> ...so this should be configurable.. logger.add(Logger::DEBUG) { "Rendered..." } would be fine

Could you please clarify?
I follow the same idea of [Active Job](https://github.com/rails/rails/blob/2ad9afe4ff2c12d1a9e4a00d9009d040e636c9b0/activejob/lib/active_job/logging.rb#L56) and [Action View](https://github.com/rails/rails/blob/2ad9afe4ff2c12d1a9e4a00d9009d040e636c9b0/actionview/lib/action_view/log_subscriber.rb#L16) for example the uses the `info` to log.